--- a/squashfs-tools/unsquashfs.c	2019-08-20 22:58:06.009563653 +0200
+++ b/squashfs-tools/unsquashfs.c.orig	2019-08-20 22:57:54.473615591 +0200
@@ -32,6 +32,7 @@
 #include <zlib.h>
 #include <sys/mman.h>
 #include <utime.h>
+#include <sys/sysmacros.h>
 
 #ifndef linux
 #define __BYTE_ORDER BYTE_ORDER
--- a/squashfs-tools/mksquashfs.c	2019-08-20 22:56:33.834032472 +0200
+++ b/squashfs-tools/mksquashfs.c.orig	2019-08-20 22:56:21.870102928 +0200
@@ -40,6 +40,7 @@
 #include <signal.h>
 #include <setjmp.h>
 #include <sys/mman.h>
+#include <sys/sysmacros.h>
 
 #ifndef linux
 #define __BYTE_ORDER BYTE_ORDER
